views/result/account.validation.success.html

```diff
@@ -1,15 +1,41 @@
+{% from "govuk/components/button/macro.njk"  import govukButton %}
+
 {% set html %}
     <h3 class="govuk-notification-banner__heading">
         The validation of the document '{{accountValidationResult.fileName}}' has been successful
     </h3>
 
-    {% if accountValidationResult.imageUrl %}
-        <p class="govuk-body"><a class="govuk-notification-banner__link" href="{{ accountValidationResult.imageUrl }}">Download generated image.</a></p>
-    {% endif %}
 {% endset %}
 
 {{ govukNotificationBanner({
     titleText: 'Success',
     type: 'success',
     html: html
-}) }}
\ No newline at end of file
+}) }}
+
+{% block content %}
+    <h2 class="govuk-heading-m">What happens next</h2>
+    <p class="govuk-body">You can select show image to preview the image that will be published on the Companies House register</p>
+     {% if accountValidationResult.imageUrl %}
+        <div>
+            {{ govukButton({
+                text: "Show image",
+                classes: "govuk-button--secondary",
+                href: accountValidationResult.imageUrl,
+                attributes:{
+                    "target":"_blank"
+                }
+            }) }} 
+        </div>
+    {% endif %}
+    
+    <p class="govuk-body">
+        You can <a href={{BASE_URL}}>validate another single file or zip file.</a>
+    </p>
+    <p class="govuk-body">
+        You can return to <a href={{CHS_URL}}>Companies House home page.</a>
+    </p>
+
+{% endblock %}
```

Comment:
We don't need the content block here as it is already in the [result view](https://github.com/companieshouse/account-validator-web/blob/729002e8d8687f6ce8b6d5ebe0d669a09896bf43/views/result/result.html#L18). The different validation results are then included in the result view with the [`showValidationResult()`](https://github.com/companieshouse/account-validator-web/blob/729002e8d8687f6ce8b6d5ebe0d669a09896bf43/views/result/result.html#L34).


```suggestion
    <h2 class="govuk-heading-m">What happens next</h2>
    <p class="govuk-body">You can select show image to preview the image that will be published on the Companies House register</p>
     {% if accountValidationResult.imageUrl %}
        <div>
            {{ govukButton({
                text: "Show image",
                classes: "govuk-button--secondary",
                href: accountValidationResult.imageUrl,
                attributes:{
                    "target":"_blank"
                }
            }) }} 
        </div>
    {% endif %}
    
    <p class="govuk-body">
        You can <a href={{BASE_URL}}>validate another single file or zip file.</a>
    </p>
    <p class="govuk-body">
        You can return to <a href={{CHS_URL}}>Companies House home page.</a>
    </p>
```